Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:lts-slim as build
+FROM node:lts-buster-slim as build
```

Comment:
The slim image does not contain Python or other buildpack-deps, which caused a different PR to fail because Node needs Python to build `node-gyp`, so I think we want to change to the full `buster` image.

I would like us to pin the node version here. We've seen that builds fail randomly because we have used the LTS tag when the LTS version changes under our feet.

Now I have verified that everything works with node 14, so I suggest changing the line to:

`FROM node:14-buster as build`

Reply:
Yep makes sense.